web/pdfjs.js

```diff
@@ -21,10 +21,14 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('PRODUCTION')) {
   var pdfjsLib;
   // The if below protected by __pdfjsdev_webpack__ check from webpack parsing.
   if (typeof __pdfjsdev_webpack__ === 'undefined') {
-    if (typeof require === 'function') {
-      pdfjsLib = require('../build/pdf.js'); // using a bundler to pull the core
-    } else {
-      pdfjsLib = window['pdfjs-dist/build/pdf']; // loaded via html script tag
+    if (typeof window !== 'undefined' && window['pdfjs-dist/build/pdf']) {
+      pdfjsLib = window['pdfjs-dist/build/pdf'];
+    }
+    else if (typeof require === 'function') {
```

Comment:
As you can see in https://travis-ci.org/mozilla/pdf.js/builds/210076791#L501, this causes ESLint failures. Please note that the coding style calls for `}` and `else` to be placed on the same line, i.e. `} else if (...` (and similar below).